### DISCUSSION TITLE
Why writing screen name 2 time? when using with observer?

### DISCUSSION THREAD

**Discussion by mrshahzeb7** [infinitered/ignite#2576] [General]
ignite Code:
export const ActivePlans: ActivePlansProps = observer(function ActivePlans(
  _props
) {
return <View style={styles.container} />
})

my updated code
export const ActivePlans: ActivePlansProps = observer((_props) => {
  return <View style={styles.container} />
})

is there any reason for it?

**Comment by frankcalise** [comment]
Hi @mrshahzeb7, it is related to the `displayName` not being set on anonymous components(which you could set directly on `ActivePlans` in your example).  So it's helpful for debugging and such

https://github.com/infinitered/ignite/discussions/2346

**Reply by mrshahzeb7** [reply]
Thankyou 👍
